benchmarks/benchmark_latency.py

```diff
@@ -105,7 +109,9 @@ def run_to_completion(profile: bool = False):
         'for BF16 models.')
     parser.add_argument(
         '--profile',
-        action='store_true',
-        help='profile the generation process of a single batch')
+        type=str,
+        default=None,
+        help=('path to save the pytorch profiler output. If unspecifed, '
+              'no profiling is done.'))
```

Comment:
Let's create a new argument `--profile-result-dir` and if unspecified, set it to something like `vllm_benchmark_result/latency_result_[date]_[time]/`? This should make the naming of multiple profiling runs simpler.

Reply:
Nit: And also add `the result can be visualized on https://ui.perfetto.dev/` into the help message.